pkg/api/index.go

```diff
@@ -171,10 +171,6 @@ func (hs *HTTPServer) getNavTree(c *models.ReqContext, hasEditPerm bool) ([]*dto
 	}
 
 	dashboardChildLinks := hs.buildDashboardNavLinks(c, hasEditPerm)
-	inventoryChildNavs := []*dtos.NavLink{
```

Comment:
Indeed. Before, we declared some routes on the backend. But because this route might not be available for everyone, all Percona's routes are defined on the Navbar, on the `buildInventoryAndSettings` function.